python/tk_nuke/menu_generation.py

```diff
@@ -44,19 +42,17 @@ def create_menu(self):
         """
         Render the entire Shotgun menu.
         """
-        # create main menu
-        nuke_menu = nuke.menu("Nuke")
-        menu_handle = nuke_menu.addMenu(self._menu_name) 
+        # create main Shotgun menu
+        menu_handle = nuke.menu("Nuke").addMenu(self._menu_name)
         
-        # the right click menu that is displayed when clicking on a pane 
-        self._pane_menu = nuke.menu("Pane") 
         # create tank side menu
-        node_menu_handle = nuke.menu("Nodes").addMenu(self._menu_name, icon=self.tank_logo)
+        node_menu_handle = nuke.menu("Nodes").addMenu(self._menu_name, icon=self._shotgun_logo)
```

Comment:
Some stuff is centralized - for example the standard banner on top of the UI dialogs. In the case of engines, logos and resources are often pretty specific and selected specifically to suit the needs of a particular engine. I think this is the reason we haven't tried to standardize things. I think it's better to wait with standardizing any icons until we do a big menu overhaul. That way we'll have a better, clearer picture of all the different moving pieces we need to take into account for that work.